publishing.gradle

```diff
@@ -156,10 +156,17 @@ afterEvaluate { project ->
         }
 
         task androidJavadocs(type: Javadoc) {
+            failOnError false
```

Comment:
What happens when it fails then?

Reply:
The Javadocs still generate, I have a bit more detail in the PR description.  Many of the issues are due to an increase in the strictness of the HTML parser, when browsers support the features (auto-closed <br/> tags, nested tags, <code> tags at all, etc.).  To do a full audit of the issues would take much more time given their quantity.